people/src/hog.cpp

```diff
@@ -41,12 +41,12 @@
 
 #include <pcl/people/hog.h>
 
-#include <cstring>
```

Comment:
`cstring` needed for `memcpy`

Reply:
Heh. So that's not explicit required on windows. I'll add it again. Thanks